lib/resources/checks.js

```diff
@@ -1,18 +1,14 @@
 'use strict';
 
-const util         = require('util');
 const ResourceBase = require('./resourceBase');
 
-const Checks = function (config) {
-  ResourceBase.call(this, 'checks', config);
-};
-
-util.inherits(Checks, ResourceBase);
-
-(function () {
+module.exports = class Checks extends ResourceBase {
```

Comment:
nit: We want to try and keep all exports at the bottom.

That said, I'm not sure if that means you need to set this class equal to this, or if you can just have the following at the bottom.

```js
module.exports = Checks;
```